CHANGELOG.md

```diff
@@ -7,6 +7,15 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 
 ## Unreleased
 
+### Bugfixes
+
+#### node
+
+- A transitive dependency used submodule exports, which aren't supported yet by
+significant parts of the ecosystem, such as Jest. With an internal change, we enabled
+using @inrupt/solid-client-authn-node without modifying Jest configuration of
+our dependents. 
```

Comment:
Since there are more reasons than just Jest why this could be problematic, I'd stick to only mentioning it as an example of what can go wrong.

```suggestion
using @inrupt/solid-client-authn-node without encountering submodule exports.
```